src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im.contexts.wallet.send.from.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.floating-button-page.view :as floating-button-page]
+    [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
+    [status-im.contexts.wallet.send.from.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn on-press
+  [address]
+  (rf/dispatch [:wallet/select-from-account
+                {:address  address
+                 :stack-id :screen/wallet.select-from}]))
+
+(defn view
+  []
+  (let [on-close (fn []
```

Comment:
Just to be on the safe side, I would declare it outside the function body, so will not trigger any re-render

Reply:
Done